task/backend/executor/task_executor.go

```diff
@@ -407,6 +405,22 @@ func (w *worker) executeQuery(p *Promise) {
 	w.finish(p, backend.RunSuccess, runErr)
 }
 
+// RunsActive returns the current number of workers, which is equivalent to
+// the number of runs actively running
+func (e *TaskExecutor) RunsActive() float64 {
+	return float64(len(e.workerLimit))
```

Comment:
For this one I would do the float conversion in the metrics system. Runs active (from the taskExecutor) should be a int or int64.